File: uri 1115 - Quadrant.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int x,y;
    do{
        scanf("%d %d",&x,&y);
        if(x!=0 && y!=0){
            if(x>0.0 && y>0.0)
                printf("primeiro\n");
            else if(x>0.0 && y<0.0)
                printf("quarto\n");
            else if(x<0.0 && y>0.0)
                printf("segundo\n");
            else if(x<0.0 && y<0.0)
                printf("terceiro\n");
            }
    }while(x!=0 && y!=0);
    return 0;
}

File: uri 1115 - Quadrant.py
while True:
    a, b =map(int, input().split(" "))
    if a == 0 or b == 0:
        break
    elif a > 0 < b:
        print("primeiro")
    elif a > 0 > b:
        print("quarto")
    elif a < 0 > b:
        print("terceiro")
    else:
        print("segundo")
